frontend/packages/kubevirt-plugin/src/models/templates/vm-template-yaml.ts

```diff
@@ -8,14 +8,11 @@ apiVersion: ${TemplateModel.apiGroup}/${TemplateModel.apiVersion}
 kind: ${TemplateModel.kind}
 metadata:
   name: vm-template-example
-  namespace: default
   labels:
     template.kubevirt.io/type: vm
     os.template.kubevirt.io/fedora31: 'true'
-    flavor.template.kubevirt.io/tiny: 'true'
+    flavor.template.kubevirt.io/Custom: 'true'
     workload.template.kubevirt.io/server: 'true'
-    vm.kubevirt.io/template: fedora-server-tiny-v0.7.0
```

Comment:
removed the pointers to the common templates as this yaml should get used only when the common templates get missing